testing/new-buildbot-worker.rst

```diff
@@ -195,6 +195,13 @@ by tests that fail.  Unfortunately we do not currently have a way to notify you
 only of failures on your builders, so doing periodic spot checks is also a good
 idea.
 
+If your buildbot worker is disconnecting regularly, it may be a symptom of the
+default keep-alive value being set too high (``600`` for 10 minutes). You
+can edit the `**keepalive** setting
+<https://docs.buildbot.net/latest/manual/installation/worker.html#cmdoption-buildbot-worker-create-worker-keepalive>`_
+in the ``buildbot.tac`` file found in your build area to a lower value
+(e.g. ``180`` for 3 minutes).
+
```

Comment:
```suggestion
.. note::
   If your buildbot worker is disconnecting regularly, it may be a symptom of the
   default ``keepalive`` value (``600`` for 10 minutes) being `set too high
   <https://docs.buildbot.net/latest/manual/installation/worker.html#cmdoption-buildbot-worker-create-worker-keepalive>`_
   You can change it to a lower value (e.g. ``180`` for 3 minutes)
   in the ``buildbot.tac`` file found in your build area.

```
I added a `.. note::` to make this more visible, moved the link to `set too high` (since the link explains that), and rephrased accordingly.  (I haven't tested this.)